### PR TITLE
Use `nbsphinx` to convert Jupytyer notebooks to Sphinx website

### LINKED ISSUE #26: Tutorial Examples
Turn the code blocks given in tutorials into Python scripts (and then reference them in the documentation), so it can be easily downloaded and executed. It is also important to check that they run properly as the code changes.

Example: http://theochem.github.io/horton/2.1.1/user_postproc_aim.html#python-interface-to-the-partitioning-code

### PR DESCRIPTION
**Pull request by fwmeng88**
In this PR, [`nbsphinx`](https://github.com/spatialaudio/nbsphinx/) is used to host Jupyter notebook files for sphinx website. Main changes include

- [x] Delete RST files and use notebooks instead
- [x] Add Python syntax highlighting and add copy button
- [x] Add binder badge to link to the interactive notebook
- [x] Update configuration file, `conf.py`
- [x] Fix equations in notebooks

Sorry for this PR as it's larger than unusual and maybe a little difficult to review.  Therefore, I created a preview at https://procrustes-test.readthedocs.io/en/latest/.

[Edit] : I don't know if we need to have MyBinder badge to link the jupyter notebooks in the tutorial summary page. If you want to delete it, I can make one more commit to fix this.  https://procrustes-test.readthedocs.io/en/latest/usr_doc_tutorials.html 

This should fix #26.
Thanks! @FarnazH @PaulWAyers @Ali-Tehrani

### CODE REVIEW

File: docs/notebooks/Chirality_Check.ipynb [hunk 1]
```diff
@@ -157,9 +157,9 @@
...
       "text/plain": [
-       "<Figure size 720x576 with 1 Axes>"
+       "<Figure size 576x432 with 1 Axes>"
```
Comment (Ali-Tehrani) [inline docs/notebooks/Chirality_Check.ipynb:162]: I can't comment outside of this, but "$ 4.43 :nbsphinx-math:`times `10^{-8} Å$;" is not being rendered and in "label1="enantiomer1 with orthoation and reflection"," orthoation should be "rotation and reflection" or "orthogonal".

**Review by Ali-Tehrani — COMMENTED** [review]
I think it looks great. 

I don't know if this is already known but I couldn't install it from "pip install qc-procrustes" after doing "conda create -n myenv python=3.6", activating it and "conda install scipy".  It says 
"ERROR: Could not find a version that satisfies the requirement qc-procrustes (from versions: none)
ERROR: No matching distribution found for qc-procrustes".

Also, "conda install -c theochem procrustes" didn't work either, it couldn't find the package.  I don't know much about pip or conda to fix these.

**Comment by fwmeng88** [comment]
> I think it looks great.
> 
> I don't know if this is already known but I couldn't install it from "pip install qc-procrustes" after doing "conda create -n myenv python=3.6", activating it and "conda install scipy". It says
> "ERROR: Could not find a version that satisfies the requirement qc-procrustes (from versions: none)
> ERROR: No matching distribution found for qc-procrustes".
> 
> Also, "conda install -c theochem procrustes" didn't work either, it couldn't find the package. I don't know much about pip or conda to fix these.

We haven't released our package in PyPi or Conda. So you got all the errors. Should be OK after releasing our package.

File: docs/notebooks/Chemical_Structure_Alignment.ipynb [hunk 2]
```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Molecular alignment is a fundamental problem in cheminformatics and can be usedfor  structure  determination,  similarity  based  searching,  and  ligand-baseddrug design et al. This problem can be solved by orthogonal Procrustes when given two matrices representing three-dimensional coordinates.  The code block below showsthe ease-of-use of the `Procrustes` library for protein structure alignment, one of themost fundamental problems in structural biophysics."
+    "Molecular alignment is a fundamental problem in cheminformatics and can be usedfor  structure  determination,  similarity  based  searching,  and  ligand-baseddrug design et al. This problem can be solved as a orthogonal Procrustes problem when given two matrices representing three-dimensional coordinates.  The code block below showsthe ease-of-use of the `Procrustes` library for protein structure alignment, one of themost fundamental problems in structural biophysics."
```
[the remark this answers is not in the document]
Comment (fwmeng88) [reply docs/notebooks/Chemical_Structure_Alignment.ipynb]: Thanks for noticing this! It has been fixed in ced0538.

File: docs/notebooks/Chirality_Check.ipynb (hunk 1 above)
Comment (fwmeng88) [reply docs/notebooks/Chirality_Check.ipynb:162]: This is fixed in ced0538.
Thanks for reviewing carefully.

File: docs/notebooks/Chemical_Structure_Alignment.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ali-Tehrani) [reply docs/notebooks/Chemical_Structure_Alignment.ipynb]: Thanks for fixing this, there still isn't a space in "ligand-baseddrug", and "themost" within the same paragraph.

**Review by FarnazH — APPROVED** [review]

**Pull request merged**